release/release_gold.sh

```diff
@@ -96,6 +97,10 @@ function main {
 
 	lc_time_run test_boms
 
+	lc_background_run clone_repository liferay-portal-ee
+
+	lc_wait
+
```

Comment:
Include the function [clean_repository](https://github.com/liferay/liferay-docker/blob/fdd1136d57c8c1c578fb102c1dd6f7340311a7a8/release/_git.sh#L3-L15) to prevent issues manipulating the git repository and its branches.